postman/grants-ui-backend.postman_collection.json

```diff
@@ -0,0 +1,41 @@
+          "options": {
+            "raw": {
+              "language": "json"
+            }
+          }
+        },
+        "url": {
+          "raw": "{{grants-ui-backend_base_url}}/save",
```

Comment:
Something more descriptive could be better like /state/save or /application/state/save. The API is more self-documenting then